src/pages/MyArea/MyArea.tsx

```diff
@@ -3,14 +3,20 @@ import { MyAreaHeader, MyAreaMap } from 'components/MyArea/MyArea';
 import React, { useContext } from 'react';
 
 import styles from './MyArea.module.scss';
+import { usePhoneScreen } from 'hooks/media.hook';
 
 export default () => {
-  const {
+  let {
     MY_AREA: {
       url: { advanced: mapUrl },
     },
   } = useContext(AppContext);
 
+  // We have to figure out if we are on a phone screen on this level and not in
```

Comment:
The `useMyMap()` hook _is_ called in a component so you can add the following code there.